webapp/locales/de.json

```diff
@@ -452,6 +452,7 @@
         "filterALL": "Alle Beiträge anzeigen",
         "success": "Gespeichert!",
         "languageSelectLabel": "Sprache",
```

Comment:
In the english version it is named:
`"languageSelectLabel": "Language of your contribution“`

In german that would be:
"languageSelectLabel": „Sprache deines Beitrags“

I would like to change this, because a lot of people seem to think this is as well the language of the web app and not of the post.
But anyways. This doesn’t hold me back to approve and we can even do it in an other PR.

Reply:
https://github.com/Human-Connection/Human-Connection/issues/2583